app/src/main/java/com/jerboa/DefaultInstances.kt

```diff
@@ -0,0 +1,60 @@
+package com.jerboa
+
+val DEFAULT_LEMMY_INSTANCES = arrayOf(
+    "lemmy.world", // 13856 monthly users
+    "lemmy.ml", // 4206 monthly users
+    "beehaw.org", // 3730 monthly users
+    "feddit.de", // 2300 monthly users
+    "sh.itjust.works", // 2215 monthly users
+    "www.hexbear.net", // 1603 monthly users
+    "lemmynsfw.com", // 1122 monthly users
```

Comment:
We may want to consider excluding primarily NSFW instances from the default instances list due to play store policies.